selfdrive/car/subaru/values.py

```diff
@@ -46,4 +63,8 @@ class CAR:
   CAR.ASCENT: dbc_dict('subaru_global_2017', None),
   CAR.IMPREZA: dbc_dict('subaru_global_2017', None),
   CAR.FORESTER: dbc_dict('subaru_global_2017', None),
+  CAR.LEGACY_2015: dbc_dict('subaru_outback_2015_generated', None),
 }
+
+GLOBAL_CAR = [CAR.ASCENT, CAR.IMPREZA, CAR.FORESTER]
+LEGACY_CAR = [CAR.LEGACY_2015]
```

Comment:
Let's call it `PREGLOBAL_CARS`, and we only need one list (the global cars are just all the subarus that aren't pre-global).

Reply:
done